### ISSUE TITLE
Local Run Unable to Export Code after update to version 5.0.8 on October 22nd

### ISSUE THREAD

**Issue by richenri** [FlutterFlow/flutterflow-issues#4613]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

After updating to version 5.0.8, hot reload fails to export the code, not reflecting the changes made and the IDE shows that it is not up to date. I tried to create a new project to validate if there was a problem with my project or dependencies, but the same problem occurred.
![image](https://github.com/user-attachments/assets/5f099811-cd0a-461e-b6b9-d445faecbd7f)

### Expected Behavior

Reflect my changes

### Steps to Reproduce

1. Run Local Run 
2. Make some changes
3. Apply the Hot reload

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT40lsmBw5dgrs9J1oSJbsFFimU5Jm16R+cVjup7aCosfZT2BLMcec/RVERtOe2+SkFPfGCWkWoW0tHHkof9EPYoPSmWftVAzrlcSDrKbHy7RLKLFJWgOnR8BZ9jJ2q+35+33RBANfhtdGQC3DmMHNiydDLvQee2OjA8Da/LZO4=

### Visual documentation

![image](https://github.com/user-attachments/assets/45bc18d3-c522-467c-b6c2-9265bf5c6d78)

### Environment

```markdown
- FlutterFlow version: 5.0.8
- Platform: Windows
- Browser name and version: Edge
- Operating system and version affected: Windows 11
```

**Comment by cmregazzo** [comment]
Goodnight,

I have the same problem since updating to FF version 5.0.8.

Tks.,

**Comment by Alezanello** [comment]
Hello!

I wasn’t able to export your code either. I’ll reach out to the engineering team to help identify the root of this issue—thank you for your patience!

@cmregazzo, could you please provide a Bug Report Code here so we can also check your project?

**Comment by laughingbuddha24** [comment]
I have this issue too

**Comment by VossApps** [comment]
I have also been experiencing this problem since the update to version 5.0.8 on the Windows Desktop app.

I would like to describe the issue in more detail for the support team.

Errors:

1. When starting the local run for the first time, nothing happens, and when trying to start it the second time, the defining application launches.
2. The local app assets are not transferred, resulting in an error message (see below).
3. A hot reload leads to the error message "Failed to export code."

My troubleshooting attempts:

- Created a new project
- Completely uninstalled and reinstalled the FlutterFlow Windows Desktop app (Windows 11)
- Updated project dependencies
- Checked the compatibility of the JAVA JDK and Gradle versions
- Opened in VS Code and executed "flutter run" from there
- Installed Android Studio (Ladybug) and tested there (there also seems to be a bug since the JAVA JDK version from Android Studio is not compatible with Flutter)

*None of these attempts were successful.

We kindly request prompt support or a workaround, as currently, development is not possible.



------DEVICE LOGS----------
Launching lib\main.dart on SM S901B in debug mode...
Upgrading build.gradle
Running Gradle task 'assembleDebug'...                          
Warning: This version only understands SDK XML versions up to 3 but an SDK XML file of version 4 was encountered. This can happen if you use versions of Android Studio and the command-line tools that were released at different times.
Note: C:\Users\xxx\AppData\Local\Pub\Cache\hosted\pub.dev\vibration-2.0.1\android\src\main\java\com\benjaminabel\vibration\VibrationPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Running Gradle task 'assembleDebug'...                             72,7s
√ Built build\app\outputs\flutter-apk\app-debug.apk
Installing build\app\outputs\flutter-apk\app-debug.apk...          12,5s
Syncing files to device SM S901B...                                171ms

Flutter run key commands.
r Hot reload. 
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

A Dart VM Service on SM S901B is available at: http://127.0.0.1:57233/1Xbd7sjri1k=/
I/eapps.findbirdy(30270): [HIDL_FETCH_IMapper] android.hardware.graphics.mapper@4.0: Loaded Mapper successfully.
I/blob    (30270): initializing random seed using 81076579885736
I/blob    (30270): get: caller's buffer is too small for value: 0 (needs 11175)
I/blob    (30270): get: copying 11175 bytes to caller's buffer
I/blob    (30270): get: caller's buffer is too small for value: 0 (needs 11175)
I/blob    (30270): get: copying 11175 bytes to caller's buffer
I/BLASTBufferQueue(30270): [SurfaceView[com.simpleapps.findbirdy/com.simpleapps.findbirdy.MainActivity]@0#1](f:0,a:0,s:0) onFrameAvailable the first frame is available
D/SurfaceView@4efcb6(30270): 5176502 setAlpha: alpha=1.0
I/SurfaceView(30270): 5176502 Changes: creating=false format=false size=false visible=false alpha=true hint=false visible=false left=false top=false z=false attached=true lifecycleStrategy=false
I/SurfaceView@4efcb6(30270): 5176502 Cur surface: Surface(name=null)/@0x1eee087
I/SurfaceView@4efcb6(30270): updateSurface: mVisible = true mSurface.isValid() = true
I/SurfaceView@4efcb6(30270): updateSurface: mSurfaceCreated = true surfaceChanged = false visibleChanged = false
I/SurfaceView(30270): 5176502 surfaceRedrawNeeded
I/SurfaceView(30270): 5176502 finishedDrawing
V/SurfaceView@4efcb6(30270): Layout: x=0 y=0 w=1080 h=2196, frame=Rect(0, 0 - 1080, 2196)
The Flutter DevTools debugger and profiler on SM S901B is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:57233/1Xbd7sjri1k=/

══╡ EXCEPTION CAUGHT BY IMAGE RESOURCE SERVICE ╞════════════════════════════════════════════════════
The following assertion was thrown resolving an image codec:
Unable to load asset: "assets/images/pic1.jpg".
Exception: Asset not found

When the exception was thrown, this was the stack:
#0      PlatformAssetBundle.loadBuffer (package:flutter/src/services/asset_bundle.dart:368:7)
<asynchronous suspension>
#1      AssetBundleImageProvider._loadAsync (package:flutter/src/painting/image_provider.dart:757:16)
<asynchronous suspension>
#2      MultiFrameImageStreamCompleter._handleCodecReady (package:flutter/src/painting/image_stream.dart:1013:3)
<asynchronous suspension>

Image provider: AssetImage(bundle: null, name: "assets/images/pic1.jpg")
Image key: AssetBundleImageKey(bundle: PlatformAssetBundle#788a9(), name: "assets/images/pic.jpg",
  scale: 1.0)

**Comment by BinitKumarMahato** [comment]
I am facing the same problem after the update. (5.0.8)

**Comment by zinho81** [comment]
I am facing the same problem, too. It started after updating to 5.0.8+.

**Comment by richenri** [comment]
Problem solved in version 5.0.9. Thanks!!!

**Comment by aemelyanovff** [comment]
Fixed in the latest version of the desktop app for Windows!
